src/mame/netronics/exp85.cpp

```diff
@@ -41,24 +54,33 @@
 #include "imagedev/cassette.h"
 #include "sound/spkrdev.h"
 
+#define VERBOSE 0 //(1U)
+#include "logmacro.h"
 
 namespace {
 
-#define I8085A_TAG      "u100"
-#define I8155_TAG       "u106"
-#define I8355_TAG       "u105"
+#define I8085A_TAG       "u100"
+#define I8155_TAG        "u106"
+#define I8355_TAG        "u105"
+#define EXTENDED_RAM_TAG "extended_ram"
+
+#define LOW_MEMORY_ROM_MIRROR_ENTRY 0
```

Comment:
For numeric constants it's preferred now to use the C++ `static constexpr` form, so in this case `static constexpr int LOW_MEMORY_ROM_MIRROR_ENTRY = 0;`  That way the compiler can do better sanity checking.

Reply:
Fixed